src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
You already have the / in the api call.
```suggestion
export const BASE_URL = 'https://mate.academy/students-api';
```